pubspec.yaml

```diff
@@ -31,64 +31,66 @@ dependencies:
   flutter:
     sdk: flutter
 
+  mockito: ^5.0.16
+  build_runner: ^2.0.0
 
-  encrypt: ^5.0.3
```

Comment:
please rebase - we don't want to downgrade our packages

Reply:
rebased and made sure the versions matched